pyproject.toml

```diff
@@ -71,7 +71,7 @@ cov-ci = [
 ]
```

Comment:
Line 10 in this file should go from 
requires-python = ">=3.7"
to
requires-python = ">=3.9"

Reply:
Also see 'classifiers' line 16, which lists the old pythons and misses out python 3.12

Reply:
this whole thing is just so awkward - I have no idea why the package skeletons don't have a single source of truth that does not require listing out individual versions rather than ranges